qrenderdoc/Code/Interface/PersistantConfig.h

```diff
@@ -372,6 +372,16 @@ DECLARE_REFLECTION_STRUCT(BugReport);
       "Defaults to ``7``.");                                                                       \
   CONFIG_SETTING_VAL(public, int, int, Formatter_PosExp, 7)                                        \
                                                                                                    \
+  DOCUMENT(                                                                                        \
+      "The formatting mode to use for values marked as Offsets or Sizes.\n"                        \
+      "\n"                                                                                         \
+      "E.g. Auto: decimal by default and hexadecimal if above a certain threshold, "               \
```

Comment:
I was not sure about the formatting style to use here.
If I build documentation locally will I see these changes?

Reply:
Yes these will be documented on the `PersistantConfig`. This formatting seems fine to me.